src/test/java/de/retest/recheck/cli/testutils/TestReportCreator.java

```diff
@@ -51,7 +51,7 @@ public static String createTestReportFileWithWarnings( final TemporaryFolder fol
 		final SuiteReplayResult suite = createSuiteReplayResultWithDiffs( folder );
 
 		final ElementIdentificationWarning elementIdentificationWarning =
-				new ElementIdentificationWarning( "someTestClass", null );
+				new ElementIdentificationWarning( "FooTest", null, null, "someTestClass" );
```

Comment:
Don't know if you need this data at all, but if you pass something, I would try to get close to what is expected:

```suggestion
				new ElementIdentificationWarning( "FooTest", null, null, "com.example.FooTest" );
```